    
    # Effort and velocity limits the physical effort/velocity on the joint in Gazebo. ROS_control is not part of this, ros_command can get larger than this value. This is comparable to slip or some other mechanical limit on torque, current.
    
    joint_limits:
      leg_1:                                     #LEG. Other legs are copied from leg_1 on python script
        has_position_limits: true
        min_position: -2.0
        max_position: 0.0
        has_velocity_limits: false
        max_velocity: 10.0
        has_acceleration_limits: false
        max_acceleration: 5.0
        has_jerk_limits: false
        max_jerk: 100.0
        has_effort_limits: true
        max_effort: 50.0
      third_frame_prismatic:                      #PRISMATIC
        has_position_limits: true
        min_position: -0.6
        max_position: -0.2
        has_velocity_limits: true
        max_velocity: 0.5
        has_acceleration_limits: true
        max_acceleration: 5.0
        has_jerk_limits: true
        max_jerk: 100.0
        has_effort_limits: true
        max_effort: 50.0
      third_frame_revolute:                       #REVOLUTE
        has_position_limits: true
        min_position: -0.3
        max_position: 0.3
        has_velocity_limits: true
        max_velocity: 0.5
        has_acceleration_limits: true
        max_acceleration: 5.0
        has_jerk_limits: true
        max_jerk: 100.0
        has_effort_limits: true
        max_effort: 50.0



